### DISCUSSION TITLE
help

### DISCUSSION THREAD

**Discussion by Splerger** [ThioJoe/YT-Spammer-Purge#641] [Q&A] [answered]
how do i find the exe file one because the python version does not work for me

**Comment by Kurdakov** [comment, marked as answer]
here https://github.com/ThioJoe/YT-Spammer-Purge/releases you might download releases (the link is on the right side of main project page)

**Comment by ThioJoe** [comment]
Also see: 
<img width="677" alt="image" src="https://user-images.githubusercontent.com/12518330/153769380-29e73b8e-8fc4-42c7-835a-2f7f6992fed9.png">
